app/jobs/payout/paypal_job.rb

```diff
@@ -17,7 +17,6 @@ def perform(should_send_notifications: false, payout_report_id: nil, publisher_i
       IncludePublisherInPayoutReportJob.perform_async(
         payout_report_id: payout_report_id,
         publisher_id: publisher.id,
-        should_send_notifications: should_send_notifications,
```

Comment:
If you're going to remove, would you remove it everywhere in the codebase? The particular call-chain can be found with `should_only_notify`.

Reply:
I put it back, we should handle all this in a separate PR.